next.config.mjs

```
@@ -197,41 +245,26 @@ const nextConfig = {
   },
 };
 
+// Remove webpack config when Turbopack is active to silence warnings
+if (process.env.TURBOPACK) {
+  // Turbopack ignores Webpack config; deleting prevents noisy warning
+  delete nextConfig.webpack;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)